index.html

```diff
@@ -4,38 +4,42 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="style.css" />
     <title>mobile version</title>
   </head>
 
   <body>
-    <header class="top-head">
-      <div class="navbar">
-        <a class="navbar-container">Arrow Dan</a>
-        <a href="#">
-          <img
-            class="hamburger hide-desktop"
-            src="image/Union.png"
-            alt="navigation icon"
-          />
-        </a>
-      </div>
+    <header class="header">
+      <a class="container" href="index.html">Arrow Dan</a>
+      <nav class="navbar">
+        <img
+          class="hamburger hide-desktop"
+          src="image/Union.png"
+          alt="navigation icon"
+        />
+        <ul class="hide-mobile">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
+      </nav>
     </header>
 
     <section class="main-container container">
```

Comment:
- [ ] The headline background image is gone on the desktop view, please fix it so it will display on the desktop view and mobile view

Reply:
Hi @[Ismailco](https://github.com/Ismailco)... In the above request, In figma mobile we have humberger menu icon only, while on figma desktop, Portfolio, About and Contact appear. Kindly check.

Reply:
@dunnyk,


> Hi @[Ismailco](https://github.com/Ismailco)... In the above request, In figma mobile we have humberger menu icon only, while on figma desktop, Portfolio, About and Contact appear. Kindly check.

Yes, that's true, but you don't have the background image in the desktop view, this comment is about the background image not the nav links

#### Your design:
![Screenshot_20220428_200924](https://user-images.githubusercontent.com/25227022/165845587-70d1ebc4-5944-470c-8916-bd4421a97078.png)

#### Figma design:
![Screenshot_20220428_210718](https://user-images.githubusercontent.com/25227022/165845910-f88341fb-a553-472d-b052-67c958d05deb.png)

Reply:
ok thanks I understand now.